src/test/unit/mcmc/chains_test.cpp

```diff
@@ -925,14 +916,9 @@ TEST_F(McmcChains, blocker_split_potential_scale_reduction_rank) {
     ASSERT_NEAR(expected_tail_rhat, computed_tail_rhat, 1e-4)
         << "Tail Rhat mismatch for index: " << index
         << ", parameter: " << chains.param_name(index);
-
-    // ASSERT_NEAR(rhat(index - 4),
-    //             chains.split_potential_scale_reduction_rank(index), 1e-4)
-    //     << "rhat for index: " << index
-    //     << ", parameter: " << chains.param_name(index);
   }
 
-  for (int index = 0; index < chains.num_params(); index++) {
+  for (int index = 4; index < chains.num_params(); index++) {
```

Comment:
Why is this file changing?

Reply:
the test data are files `blocker1.csv` and `blocker2.csv` - these are CSV outputs from Stan version 1.3 with sampler params `lp__,`, `accept_stat__`, `stepsize__`, and `treedepth__`; except for `lp__`, the diagnostics will return `nan` and the test fails because `nan` != `nan`.

plus, removing commented out code